crates/viewer/re_view_spatial/src/ui_2d.rs

```diff
@@ -153,11 +150,15 @@ impl SpatialView2D {
         // Note that we can't rely on the camera being part of scene.space_cameras since that requires
         // the camera to be added to the scene!
         //
-        // TODO(jleibs): Would be nice to use `query_pinhole` here, but we don't have a data-result or the other pieces
-        // necessary to properly handle overrides, defaults, or fallbacks. We don't actually use the image_plane_distance
-        // so it doesnt technically matter.
+        // TODO(jleibs, andreas): We don't have a data-result or the other pieces
+        // necessary to properly handle overrides, defaults, or fallbacks.
```

Comment:
Is there maybe an issue for this? (pinhole-blueprint-overrides)

Reply:
I'll link the transform override issue!